install/updateVersion.sh

```diff
@@ -56,7 +56,7 @@ function gen_file() {
 }
 
 
-for target in istio.yaml istio-auth.yaml istio-one-namespace.yaml istio-one-namespace-auth.yaml;do
+for target in istio.yaml istio-auth.yaml istio-one-namespace.yaml istio-one-namespace-auth.yaml istio-multicluster.yaml istio-auth-multicluster.yaml istio-remote.yaml;do
```

Comment:
I was hoping we can get rid of updateVersion and this kind of generated files.

What is the use case ? Can you instead just have make targets, and call them when needed in tests (and also
have them generate the files in $OUTDIR, not in source tree ) ?

For the 'legacy' files it's hard to change - but for new ones we can avoid past mistakes.

Reply:
will take a look.